src/common/GenericTable/GenericTable.css

```diff
@@ -14,7 +15,7 @@
 .generic-table-container .generic-table-row {
     display: flex;
     border-bottom: solid 1px var(--brownish-grey);
-    height: 35px;
+    min-height: 35px;
```

Comment:
This change has broken the pagination on the Search Book table.

_**Note:**_ Be careful when you make style changes to a general component, always check if is rendering correctly in other places too.